environment_python_3_9.yml

```diff
@@ -21,50 +21,50 @@ channels:
   - defaults
 
 dependencies:
-  - astropy=5.0.4
-  - bokeh=2.4.2
+  - astropy=5.1
+  - bokeh=2.4.3
   - beautifulsoup4=4.11.1
-  - celery
+  - celery=4.4.0
```

Comment:
celery version doesn't match 3.8 environment of `5.0.5`

Reply:
That's because it doesn't look like any celery version >= 5 is compatible with the way we set up the 3.9 environment. I don't know why, honestly, the dependencies here are a tangle. I don't *think* I'm using any celery features that require version 5, but it's entirely possible that we are. As such, moving to python 3.9 while keeping the full dependency list, may cause problems.

Reply:
Yeah, if you look at the commit messages above, you'll see that I started with 5.0.5 and backed down one version at a time. 4.4.0 was the latest version that worked.

Reply:
Okay, that is strange. I am a little worried about using 2 different versions in two different environments. We normally would just default to the minimum working version for all of our dependency files. @bhilbert4 if @york-stsci doesn't think that version>=5 is necessary, should we just keep version `4.4.0` as the working version for all of our files?

Reply:
I guess it depends on how sure we are that we're not using any features requiring version >=5. Is there an easy way to check that? My thought is that our 3.8 environment is the most important one here, since that's what we'll have on the servers. I just want to make sure we don't break that one just for the sake of making it agree with the 3.9 environment.

Reply:
I can run a test (and, connected to that, I can put together my own PR that updates the celery cheatsheet to cover the quick process of making a celery environment for testing.

Reply:
I hear you, this shouldn't be an issue we should have to deal with (pain) but I just want to make sure we aren't "looking too far ahead". For instance, if someone is developing with version>=5 locally and then we are running a version<=5 on the servers, there could be conflicts and cause crashes. As for `redis`, should we just resort to the lowest common version? How different are the versions?

Reply:
I'm currently running some testing, and looks good so far with celery 4.4 and redis 5.0.3. I *think* that we want redis >=5 if we possibly can, but I don't remember why that's in my brain.